### PR TITLE
Implement QMK's IAP default keybind

### PR DESCRIPTION
**Pull request by jmarmstrong1207**
By default, FN2+ESC makes the keyboard go into IAP.

### CODE REVIEW

**Comment by kdarkhan** [comment]
QMK uses some standard keymap for going to DFU mode using Bootmagic.
Check [here](https://beta.docs.qmk.fm/tutorial/newbs_flashing#put-your-keyboard-into-dfu-bootloader-mode).
I wonder if there is a way to hookup some callback into that `LShift + RShift + B` such that it calls your handler and it would work in keymap independent way :thinking:

**Comment by jmarmstrong1207** [comment]
Keybinds successfully switched to the QMK default

File: keyboards/annepro2/c18/rules.mk [hunk 1]
```diff
@@ -36,7 +36,7 @@ EXTRAKEY_ENABLE = yes
 KEY_LOCK_ENABLE = no
 
 # Other featues
-BOOTMAGIC_ENABLE = no
+BOOTMAGIC_ENABLE = yes
```
Comment (Codetector1374) [inline keyboards/annepro2/c18/rules.mk:39]: Pretty sure you don't need to enable this but it's fine

**Review by Codetector1374 — APPROVED** [review]
LGTM. Please fix the shine style. We are almost good to go.

File: keyboards/annepro2/c18/rules.mk (hunk 1 above)
Comment (kdarkhan) [reply keyboards/annepro2/c18/rules.mk:39]: Right, `COMMAND_ENABLE = yes` on line 41 handles it. There is some overlap between shortcuts. On my local fork I had `COMMAND_ENABLE` disabled and that's why suggested `BOOTMAGIC_ENABLE` change.

**Pull request merged**